js/api/Forms.js

```diff
@@ -77,6 +77,7 @@ export function clearCachedForms(surveyId) {
 // Loads Form data from a single Survey and retuns it via callback after the related questions have also been fetched.
 export function loadForms(survey, callback) {
   clearCachedForms(survey.id)
+  console.log(survey)
```

Comment:
This var my be a be on the large side to dump, as it will be serialized/deserialized on the native logger.  I'm not a big fan of leaving these statements in our master branch, but haven't been saying much about the topic.  So I guess I'm starting to bring this up more now...  Furthermore, it a realm object so the Javascript debugger may work really hard with RCP calls back to the simulator/emulator.